src/snowflake/snowpark/modin/pandas/base.py

```diff
@@ -1183,10 +1183,13 @@ def between_time(
         )
         return self.loc[indexer] if axis == 0 else self.loc[:, indexer]
 
-    @base_not_implemented()
     def bfill(
-        self, axis=None, inplace=False, limit=None, downcast=None
-    ):  # noqa: PR01, RT01, D200
+        self,
+        axis: Axis | None = None,
+        inplace: bool = False,
+        limit: int | None = None,
+        downcast: dict | None = None,
+    ):
```

Comment:
Currently pandas 2.1.4 gives a `FutureWarning: DataFrame.backfill/Series.backfill is deprecated. Use DataFrame.bfill/Series.bfill instead` when trying to use backfill. Is this warning added here as well already? 

Or since backfill is deprecated, should we add it add all to our dataframe/series.rst?

Reply:
Good catch. I added the warning for both `backfill` and `pad`.